@here/harp-mapview/lib/ScreenCollisions.ts

```diff
@@ -104,6 +232,35 @@ export class ScreenCollisions {
     isVisible(bounds: Math2D.Box): boolean {
         return this.screenBounds.intersects(bounds);
     }
+
+    /**
+     * Transfer from screen space to screen bounds space
```

Comment:
What is bounds space? Please add some description.
Is there any reason why we need this "bounds" space? Can we not do everything in screen space?

Reply:
I think it is more optimal to do it in bounds space, it is just the modelViewProjection applied to some point (to get NDC) and then multiplied by the width / height, it avoids the extra addition and flip you need to do with the y. to get into screen coordinates.

I didn't change it because it would mean changing the entire text engine to move from this bounds space to screen space, which is outside of this ticket.

Reply:
I have added a comment to make it clearer.